tools/install-powershell.ps1

```diff
@@ -117,7 +117,7 @@ try {
         } elseif ($IsLinuxEnv) {
             "powershell-linux-x64"
         } elseif ($IsMacOSEnv) {
-            "powershell-osx.10.12-x64"
+            "powershell-osx-x64"
```

Comment:
@TravisEz13 Does this renaming require packaging script changes?

Reply:
The package is already named like this.  This is an unrelated bug fix.